### ISSUE TITLE
Custom helpers

### ISSUE THREAD

**Issue by Ceyar** [Sebastian-Fitzner/mangony-hbs-helpers#9]
Hi Sebastian… me again 😄 

I tried to make my custom helpers but couldn't make it work.

I'm trying to make a helpers like this:

```
Handlebars.registerHelper('trimString', function(passedString) {
    var theString = passedString.substring(0,150);
    return new Handlebars.SafeString(theString)
});
```

I tried to register the helpers in app.veams.js. Is there a specific way to register a helpers in Veams

Thank you 👋

**Comment by Sebastian-Fitzner** [comment]
I guess you want to register it in mangony to use it in your templates? If so, it has nothing to do with `veams.app.js`. 
The `mangony.config` contains an option called `helpers`: 

```
helpers: [
    'app/shared/utilities/template-helpers/*.js'
]
```

Here you can specify the path to your custom helpers. The only remaining thing is to export your helper with `module.exports.register` like so: 

``` js
module.exports.register = function (Handlebars) {
	Handlebars.registerHelper('trimString', function(passedString) {
	    var theString = passedString.substring(0,150);
	    return new Handlebars.SafeString(theString)
	});
};
```
You can see that I just wrapped your helper with another function. Hope that helps! Btw, there is already a helper provided by Mangony you can use for handling trimming of strings, see: 
- https://github.com/helpers/handlebars-helpers#truncate
- https://github.com/helpers/handlebars-helpers#truncatewords
- https://github.com/helpers/handlebars-helpers#ellipsis

**Comment by Ceyar** [comment]
Oh, great!! Thank's a lot. I'll share this to the team.

Thank you 
Have a nice day

**Comment by Sebastian-Fitzner** [comment]
May I ask you for which company you are working?

**Comment by Ceyar** [comment]
Yes sure, I’m a new employee at Campfire <https://campfire.ch/>
<https://campfire.ch/> in Switzerland.

Best regards

On 27 January 2020 at 08:17:08, Sebastian Fitzner (notifications@github.com)
wrote:

May I ask you for which company you are working?

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/Sebastian-Fitzner/mangony-hbs-helpers/issues/9?email_source=notifications&email_token=AALFYAS676DW365NKT5TZL3Q72C7JA5CNFSM4KLGVOM2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJ6QZKI#issuecomment-578620585>,
or unsubscribe

**Comment by Sebastian-Fitzner** [comment]
> Yes sure, I’m a new employee at Campfire <https://campfire.ch/> <https://campfire.ch/> in Switzerland.

Ah, I thought you started at my old employer ;) But I remember that Pius Caduff wrote a few times at Github...

**Comment by Ceyar** [comment]
Hello @Sebastian-Fitzner, I hope you are doing well!

I come back with a question about Handlebars Helpers. I tried to implement a new Helpers like you said above but I can't make it work. I get an error `Error: Missing helper: "example"`.

I added a test.js file in the `template-helpers` folder and called it in the `mangony.config.js` like this:

![Screenshot 2021-03-26 at 10 18 04](https://user-images.githubusercontent.com/1465346/112610338-dcd35d00-8e1c-11eb-802e-aa357d4d2f19.png)

In my test.js file, I added a simple example:
```
module.exports.register = function (Handlebars) {
	Handlebars.registerHelper('example', function(passedString) {
		return passedString;
	});
};
```

And then in my template I should use this example like this:

```
{{example "Some text"}}
```

Is that correct? But it doesn't work. Maybe I have to import Handlebars in my test.js file?
I am a beginner using Helpers and Veams together.

Thank you for your help
